openslides_backend/action/actions/meeting/update.py

```diff
@@ -144,3 +153,79 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         if meeting_check:
             assert_belongs_to_meeting(self.datastore, meeting_check, instance["id"])
         return instance
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        # group A check
+        if any([field in instance for field in meeting_settings_keys]) and not has_perm(
+            self.datastore,
+            self.user_id,
+            Permissions.Meeting.CAN_MANAGE_SETTINGS,
+            instance["id"],
+        ):
+            raise MissingPermission(Permissions.Meeting.CAN_MANAGE_SETTINGS)
+
+        # group B check
+        if "present_user_ids" in instance and not has_perm(
+            self.datastore, self.user_id, Permissions.User.CAN_MANAGE, instance["id"]
+        ):
+            raise MissingPermission(Permissions.User.CAN_MANAGE)
+
+        # group C check
+        if (
+            "reference_projector_id" in instance
+            or any(
+                [field for field in instance if field.startswith("default_projector")]
+            )
+        ) and not has_perm(
+            self.datastore,
+            self.user_id,
+            Permissions.Projector.CAN_MANAGE,
+            instance["id"],
+        ):
+            raise MissingPermission(Permissions.Projector.CAN_MANAGE)
+
+        # group D check
+        if any(
+            [
+                field in instance
+                for field in [
+                    "jitsi_domain",
+                    "jitsi_room_name",
+                    "jitsi_room_password",
+                    "url_name",
+                    "enable_anonymous",
+                ]
+            ]
+        ):
+            meeting = self.datastore.get(
+                FullQualifiedId(self.model.collection, instance["id"]),
+                ["admin_group_id"],
+            )
+            user = self.datastore.get(
+                FullQualifiedId(Collection("user"), self.user_id),
+                [f"group_${instance['id']}_ids"],
+            )
+            if meeting.get("admin_group_id") not in user.get(
+                f"group_${instance['id']}_ids", []
+            ):
+                raise PermissionDenied("Missing permission: Not admin of this meeting")
+
+        # group E check
+        if "organisation_tag_ids" in instance:
+            meeting = self.datastore.get(
+                FullQualifiedId(self.model.collection, instance["id"]), ["committee_id"]
+            )
+            cml_field = f"committee_${meeting['committee_id']}_management_level"
+            user = self.datastore.get(
+                FullQualifiedId(Collection("user"), self.user_id), [cml_field]
+            )
+            has_cml_permission = user.get(cml_field) == "can_manage"
```

Comment:
see https://github.com/OpenSlides/openslides-backend/pull/711#discussion_r633275788

Reply:
Use the enum here.